modules/core/shared/src/main/scala/eu/timepit/refined/numeric.scala

```diff
@@ -117,6 +120,14 @@ object numeric extends NumericInference {
         Modulo(wn.fst, wo.fst)
       )
   }
+
+  object NotNaN {
+    implicit def floatNotNaNValidate: Validate.Plain[Float, NotNaN] = fromIsNaN(_.isNaN)
+    implicit def doubleNotNaNValidate: Validate.Plain[Double, NotNaN] = fromIsNaN(_.isNaN)
+
+    def fromIsNaN[A](isNaN: A => Boolean): Validate.Plain[A, NotNaN] =
+      Validate.fromPredicate(x => !isNaN(x), x => s"$x != NaN", NotNaN())
```

Comment:
There should be parenthesis in the string representation of this predicate.
```suggestion
      Validate.fromPredicate(x => !isNaN(x), x => s"($x != NaN)", NotNaN())
```
Other predicates do this, too.